src/Iframe.tsx

```diff
@@ -109,7 +109,19 @@ function Iframe(props: IframeProps) {
     }
   }, [displayed._rev, reload?.revision])
 
-  // Set initial URL and refresh on new revisions
+  // Reload on change of URL
+  useEffect(() => {
+    if (typeof url === 'string') setDisplayUrl(url)
+  }, [url])
+
+  useEffect(handleReload, [displayUrl])
```

Comment:
Using the callback hook is more elegant, I agree. But shouldn’t `displayUrl` then be defined as dependency of the callback, so that the effect hook can be removed?